setup.py

```diff
@@ -1,47 +1,14 @@
-import os
+#!/usr/bin/env python
 import sys
 
+from setuptools import setup
 import versioneer
 
+SETUP_REQUIRES = ['setuptools >= 30.3.0']
+SETUP_REQUIRES += ['wheel'] if 'bdist_wheel' in sys.argv else []
```

Comment:
Naive question- does this work as well as a `pyproject.toml`?

Reply:
In the most recent project I set up that uses a `pyproject.toml` I still kept this low-level `setup.py` because I ran into a few issues—especially with `versioneer.py`—when working with editable/developer installs. (Though I might have been doing things incorrectly, so if you can make the `setup.py` even _more_ stripped down without this kudos—but you definitely need at least a small one for editable/developer installs.)

Reply:
Ah, okay. TBH installation stuff is the worst part of Python, so I just try to do whatever Chris M. says in his annual Gist posts. If there are tricks that need to be kept in to work, then I'm totally cool with that. I mostly use `pyproject.toml` nowadays for `black` and `isort` configuration, which probably won't be necessary here.